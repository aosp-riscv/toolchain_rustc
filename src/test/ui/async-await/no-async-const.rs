// compile-fail
// edition:2018
// compile-flags: --crate-type lib

<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
#![feature(async_await)]

=======
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
pub async const fn x() {}
//~^ ERROR expected one of `fn` or `unsafe`, found `const`
