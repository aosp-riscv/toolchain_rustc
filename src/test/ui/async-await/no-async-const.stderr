error: expected one of `fn` or `unsafe`, found `const`
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
  --> $DIR/no-async-const.rs:7:11
=======
  --> $DIR/no-async-const.rs:5:11
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
   |
LL | pub async const fn x() {}
   |           ^^^^^ expected one of `fn` or `unsafe` here

error: aborting due to previous error

